backends/arm/operators/op_rescale.py

```diff
@@ -46,13 +46,20 @@ def define_node(
         input_zp = cast(int, node.args[3])
         output_zp = cast(int, node.args[4])
 
-        if input_dtype != map_dtype(torch.int8, self.tosa_spec) and input_zp != 0:
+        if (
```

Comment:
I think 8 -> 16 and 16 -> 8 both are supported for the INT profile (and INT16)? Can you make sure you check?